cloud_governance/policy/common_policies/send_aggregated_alerts.py

```diff
@@ -142,34 +141,33 @@ def __update_delete_days(self, policy_es_data: list):
                 if not days:
                     days = 0
                 alert_user = True if self.__alert_dry_run else False
-                delete_date = 'dry_run=yes'
-                if not record.get('Skip'):
-                    record['Skip'] = 'NA'
-                if record.get('Skip') != 'NA':
+                dry_run = record.get('DryRun')
+                days_to_take_action = int(record.get('ExpireDays', self.__days_to_delete_resource))
+                if not record.get('SkipPolicy'):
+                    record['SkipPolicy'] = 'NA'
+                delete_date = ''
+                if record.get('SkipPolicy') != 'NA':
                     delete_date = 'skip_delete'
-                if self.__days_to_delete_resource - 5 == days:
+                if days_to_take_action - 5 == days:
                     delete_date = (datetime.utcnow() + timedelta(days=5)).date()
                     alert_user = True
-                elif days == self.__days_to_delete_resource - 3:
+                elif days == days_to_take_action - 3:
```

Comment:
Can you add constant instead of 5 with meaning name ?
3_RETENTATION_DAYS

Reply:
Alert admin on the last 3rd day.